docs/source/configuration/configuration_basics.md

```diff
@@ -79,13 +92,22 @@ If you want to change the way configuration is loaded, you can either [customise
 
 This section contains a set of guidance for the most common configuration requirements of standard Kedro projects:
 
-* [How to change the setting for a configuration source folder](#how-to-change-the-setting-for-a-configuration-source-folder)
-* [How to change the configuration source folder at run time](#how-to-change-the-configuration-source-folder-at-runtime)
-* [How to read configuration from a compressed file](#how-to-read-configuration-from-a-compressed-file)
-* [How to access configuration in code](#how-to-access-configuration-in-code)
-* [How to specify additional configuration environments ](#how-to-specify-additional-configuration-environments)
-* [How to change the default overriding environment](#how-to-change-the-default-overriding-environment)
-* [How to use only one configuration environment](#how-to-use-only-one-configuration-environment)
+* [Configuration](#configuration)
+  * [Configuration source](#configuration-source)
+  * [Configuration environments](#configuration-environments)
+    * [Base](#base)
+    * [Local](#local)
+  * [Configuration loading](#configuration-loading)
+    * [Configuration file names](#configuration-file-names)
+    * [Configuration patterns](#configuration-patterns)
+  * [How to use Kedro configuration](#how-to-use-kedro-configuration)
+    * [How to change the setting for a configuration source folder](#how-to-change-the-setting-for-a-configuration-source-folder)
+    * [How to change the configuration source folder at runtime](#how-to-change-the-configuration-source-folder-at-runtime)
+    * [How to read configuration from a compressed file](#how-to-read-configuration-from-a-compressed-file)
+    * [How to access configuration in code](#how-to-access-configuration-in-code)
+    * [How to specify additional configuration environments](#how-to-specify-additional-configuration-environments)
+    * [How to change the default overriding environment](#how-to-change-the-default-overriding-environment)
+    * [How to use only one configuration environment](#how-to-use-only-one-configuration-environment)
```

Comment:
There's the same problem with all the subheadings here as well, could you revert to the original list for this part?

Reply:
Reverted both.